Misc/NEWS.d/next/Core and Builtins/2022-12-11-13-22-59.gh-issue-100165.GE1Iui.rst

```diff
@@ -0,0 +1,2 @@
+Add ``BINARY_OP_AND_INT`` for and operation between int objects. Patch by
+Dong-hee Na.
```

Comment:
Just minor suggestion to not mention the exact specialised instruction, since that might change over time.
```suggestion
Specialize for and operations between ``int`` objects. Patch by
Dong-hee Na.
```